README.md

```diff
@@ -18,7 +18,7 @@ Pick a building block API (for example, pub-sub, state management) and rapidly t
 | [Secrets Management](./secrets_management/) | Securely fetch secrets |
 | Actors | Coming soon... |
 | Observability | Coming soon... |
```

Comment:
I never plan to have quickstart for observability or distributed lock.  Can i ask you to remove these Coming soons?  Actors ok to leave.